### ISSUE TITLE
Manage cached DNS

### ISSUE THREAD

**Issue by nRafinia** [AdguardTeam/AdGuardHome#2853]
Hello
Please add the ability to manage cached DNS.
* Ability to view the list of items
* Remove items from the cache list
* Search the list
* As well as a report in the form of graphs or the number of records cached in the dashboard

**Comment by ameshkov** [comment]
> As well as a report in the form of graphs or the number of records cached in the dashboard

This one is a duplicate, we're going to do that.

As for others, I'll mark it as a feature request, but to be honest, this looks kinda overkill-ish to me :)

**Comment by nRafinia** [comment]
Yes, it is a big request, but at work, if we do not have the ability to delete a record that has changed and has a high TTL, and we have to delete all cache records for one record, it is not good.

Even if there is no list of records and it is possible to delete a record in one part, I still think it is good. Such as Custom filtering -> Check the filtering

**Comment by ameshkov** [comment]
Yeah, this is a doable thing.

Maybe it'd be better to rephrase this feature request. Something like "Provide an ability to clear the DNS cache or remove individual records from it"

**Comment by nRafinia** [comment]
Yes, I agree

**Comment by uservictor** [comment]
It would be great if there were an button to clear all cached entrys.
"clear cache" or "reset"

**Comment by lizenghui** [comment]
Hope to add this feature soon, thanks
